packages/core/src/RenderPipeline/RenderQueue.ts

```
@@ -30,8 +87,16 @@
   }
 
   sortBatch(compareFunc: Function, batcherManager: BatcherManager): void {
```

Biome

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)